packages/squiggle-lang/src/rescript/Reducer/Reducer.resi

```diff
@@ -6,8 +6,21 @@ module Js = Reducer_Js
 module MathJs = Reducer_MathJs
 
 @genType
-type expressionValue = ReducerInterface_ExpressionValue.expressionValue
-
+type expressionValue = ReducerInterface.ExpressionValue.expressionValue
+@genType
+type externalBindings = ReducerInterface.ExpressionValue.externalBindings
```

Comment:
Just letting you know, genType has a hard time following modules inside modules. The way to fix `yarn bundle` failing is to change this to:
```suggestion
type expressionValue = ReducerInterface_ExpressionValue.expressionValue
@genType
type externalBindings = ReducerInterface_ExpressionValue.externalBindings
```